src/manifests/bundle_manifest.py

```diff
@@ -80,6 +83,7 @@ def __init__(self, data: Dict[str, str]):
             self.version = data["version"]
             self.platform = data["platform"]
             self.architecture = data["architecture"]
+            self.distribution: str = data.get('distribution', None)
```

Comment:
I think `data.get('distribution')` will return `None` in default if the key not exists, no need to specify it.

Reply:
It will error our in my previous setup thus I add it.
This is the same change in #1629.
Thanks.